core/dbt/adapters/base/impl.py

```diff
@@ -1298,6 +1301,57 @@ def render_column_constraint(cls, constraint: ColumnLevelConstraint) -> str:
         else:
             return ""
 
+    @available
+    def render_ddl(self, columns: Dict[str, Dict]) -> List:
+        ddl = []
```

Comment:
similar naming suggestion: `rendered_column_constraints` would be more precise here.